handbook/people-ops/travel.md

```diff
@@ -1,98 +1,166 @@
+ - Marketing conferences 
+ - Mandatory team trainings
+ -  Sales club celebrations 
+ - Quarterly Regional Sales Kickoff (Sales, Sales Ops, CE included)
 
+## [](https://about.sourcegraph.com/handbook/people-ops/travel#professional-development-and-education "#professional-development-and-education")5. Professional development and educational travel
 
-## Company-wide events
+Each teammate has $1,000 to spend on professional development, which could include attendance fees and travel to conferences, educational trainings, university classes, etc. as outlined on our  [benefits page](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks#professional-development).
 
-Sourcegraph generally holds two company-wide events each year, each fully covered by the company:
+# Budget best practices and additional spending categories
 
-- Winter party (typically in San Francisco)
-- Spring/summer event
+## Hotel accomodations:
 
-## Team events
+We advise spending within reason for the location you are visiting, with a $400 USD per night maximum spend limit. 
 
-- Each team is allocated $4,000 USD per year per teammate for team meetups (minimum two members of the team).
-- This is designed to allow each team to hold at least one event every six months (in addition to the company-wide events above).
-- This budget can be used for travel (airfare, trains, etc.), lodging (hotels, rental homes, etc.), events, meals and entertainment, and more.
-- Teams are expected to use discretion about what events are appropriate for team outings. Be inclusive! Contact @sourcegraph/people-ops with any questions.
-- Reserve plenty of time for just hanging out, touring around, having fun, etc. Quality time is important.
-- If one individual is a member of multiple teams, they will receive the same amount for each of their teams.
-- The budget can be used for online team bonding events if the team prefers.
+>  Spending over $400 USD per night requires additional approvals from  
+> Finance / People Ops. 
+> 
+> *Please send Finance/ People Ops an email with  your manager and
+> department VP cc’d including your proposal. In this   email, please
+> share screenshots from your research showing why your proposal would
+> be the most economical option given the surrounding  hotels. Finance
+> will reply with an approval or request to book other  accommodations.*
 
-## Individual travel
+## Modes of transportation:
 
-- Each teammate has one fully-paid trip per year to meet up with a group of teammates in another location.
-  - This is *in addition* to the other benefits mentioned on this page.
-  - The purpose of defining this as one “trip” (instead of providing a set dollar budget) is to avoid punishing teammates that live far away from other teammates, and who would therefore incur higher airfare costs.
+ - Economy class is recommended for booking all travel. *(If you need any higher tier of booking, reach out to Finance/People Ops for approval.)*
```

Comment:
We might not be here yet in terms of our funding story, but one policy I came to appreciate at previous employers was being allowed to book the next highest class (usually premium economy, sometimes business) for longer trips: trans-continental travel is draining (I'm Australian; ask me how I know), trips are often cut down to the minimum possible time for both personal and business reasons, and few people do well with going straight into work meetings after a sleepless, overnight 12 hour flight in economy with a 10 hour time zone difference.

If we did this, I've seen different thresholds here. Six hours was the threshold at my previous employer, which mysteriously corresponded to the length of a specific flight our CEO tended to take often. I'd suggest basing it on time zones crossed instead, with a fallback for extremely long north-south flights.

Reply:
Our Finance/People team connected and also agree we're not necessarily here yet in terms of our funding story :) Thank you for noting this, and we'll consider for future iterations of company spending revisions!